lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -50,12 +50,14 @@ export default function FilePickerApp({
         developerKey: googleDeveloperKey,
         origin: googleOrigin,
       },
+      vitalSource: { enabled: vitalSourceEnabled },
     },
   } = useContext(Config);
 
   const [activeDialog, setActiveDialog] = useState(defaultActiveDialog);
   const [url, setUrl] = useState(/** @type {string|null} */ (null));
   const [lmsFile, setLmsFile] = useState(/** @type {File|null} */ (null));
+  const [vitalSourceBook, setVitalSourceBook] = useState(false);
```

Comment:
This state is a boolean because currently the user can only choose whether the assignment uses a VS book. In this first PR they can't choose _which_ book is used.

Reply:
Perhaps a comment here to that effect?